.gitignore

```diff
@@ -21,6 +21,7 @@
 /Makefile
 /packcc
 /packcc.exe
+.cache
```

Comment:
The change for .gitignore should be in a separate commit.

Reply:
Here I add a new separate commit `gitignore: add .cache and compile_commands.json`